src/App.css

```diff
@@ -1,39 +1,9 @@
-.App {
-  text-align: center;
-}
-
```

Comment:
![image](https://github.com/Nahid1911/math-magicians/assets/93039359/bac41cc3-3249-441f-abae-c55b3e475a9d)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request professional and descriptive 😊